docs/general/glossary.md

```diff
@@ -169,11 +169,15 @@ The process of a group of entities to agree on a particular data value (such as
 makeup of blocks on a blockchain). There are a variety of algorithms used for determining consensus.
 The consensus algorithm used by Polkadot is [GRANDPA](#grandpa-finality-gadget).
 
+## Coretime
+
+Time on a relay chain core than can be purchased via Coretime system chain.
```

Comment:
Not fully true. On-demand purchases happen on relay chain. 

We already have definition for this here https://wiki.polkadot.network/docs/learn-agile-coretime#coretime 

```suggestion
The time allocated for utilizing a core that can be purchased in bulk or on demand. It is measured in relay chain blocks.
```